connectors/sync_job_runner.py

```diff
@@ -99,39 +103,42 @@ async def execute(self):
             bulk_options = self.bulk_options.copy()
             self.data_provider.tweak_bulk_options(bulk_options)
 
-            result = await self.elastic_server.async_bulk(
+            await self.elastic_server.async_bulk(
```

Comment:
I'd probably rename method to `start_async_bulk` to indicate that it starts the process but does not finish.